### PR TITLE
[CM-848] Added localisation support for last message(shown on conversation list screen)

### PR DESCRIPTION
**Pull request by Sathyan-Elangovan**
## Summary
- Added localisation support for label which will be shown on conversation list screen.(the value will be based on the last message of conversation

### CODE REVIEW

File: Sources/Views/ALKChatCell.swift [hunk 1]
```diff
@@ -299,7 +299,11 @@ public final class ALKChatCell: SwipeTableViewCell, Localizable {
         {
             messageLabel.attributedText = attributedText
         } else {
-            messageLabel.text = viewModel.theLastMessage
+            guard  let lastMessage = viewModel.theLastMessage else{
```
Comment (mukeshthawani) [inline Sources/Views/ALKChatCell.swift:302]: Format this line.

File: Sources/Views/ALKChatCell.swift [hunk 2]
```diff
@@ -299,7 +299,11 @@ public final class ALKChatCell: SwipeTableViewCell, Localizable {
...
+                messageLabel.text = ""
+                return
```
Comment (mukeshthawani) [inline Sources/Views/ALKChatCell.swift:304]: We should not return from this function if the text is not present. We need to execute below code even is the text is not present. You can use ternary operator here to check the text.

**Review by mukeshthawani — COMMENTED** [review]
Add a Changelog entry.

File: Sources/Views/ALKChatCell.swift (hunk 2 above)
Comment (Sathyan-Elangovan) [reply Sources/Views/ALKChatCell.swift]: done

File: Sources/Views/ALKChatCell.swift (hunk 1 above)
Comment (Sathyan-Elangovan) [reply Sources/Views/ALKChatCell.swift]: done

**Review by mukeshthawani — APPROVED** [review]

**Pull request merged**